README.md

```diff
@@ -6,9 +6,11 @@ I've always used SQL Sever with Entity Framework, but I recently acquired an ARM
 
 So I decided to try PostgreSQL with Entity Framework Core.
 
-Unfortunately, the PostgreSQL EF Core seems to behave differently than the SQL Server EF Core when it comes to concurrency tokens.
+This repository initially was created to investigate differences between PostgreSQL and SQL Server EF Core behavior with concurrency tokens in Domain-Driven Design scenarios.
 
-This is repository with a more minimal (I understand that this is not exactly minimal, but I wanted to keep the code as close as possible to my real project) example that reproduces the issue.
+However, after adding SQL Server implementation (PR #10), it was discovered that both PostgreSQL and SQL Server exhibit the same behavior: when adding child entities to an aggregate root, the EF change tracker appears to treat the child entity addition as a modification rather than an addition.
+
+This is repository with a more minimal (I understand that this is not exactly minimal, but I wanted to keep the code as close as possible to my real project) example that reproduces the issue with both database providers.
```

Comment:
I think this paragraphy can be rephrased.

Something like...

This repository attempts to create a minimal reproduction, but due to the complexity of the user-case, there are a lot more classes involved. Also, while using a web-application is overkill, .Net Aspire makes it easier to create integration tests involving databases.